docs/txn-what.md

```diff
@@ -1,7 +1,7 @@
 ---
 id: txn-what
 title: What are transactions?
-sidebar_label: "What are transactions?"
+sidebar_label: "Concept"
```

Comment:
For reviewers: strictly speaking, some docs in this topic do not belong to "concept". But we can rename it firstly as a low-hanging fruit and optimize it with the relevant docs later.